### PR TITLE
Small improvements in ExposureClasses documentation

### PR DESCRIPTION
**Pull request by vpnachev**
**How to categorize this PR?**

/area documentation open-source
/kind enhancement

**What this PR does / why we need it**:
Small improvements in ExposureClasses documentation
/invite @dkistner 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

**Review by dkistner — DISMISSED** [review]
/lgtm

Thank you :)

File: example/20-componentconfig-gardenlet.yaml [hunk 1]
```diff
@@ -120,11 +120,11 @@ featureGates:
 #     labels:
 #       istio: ingressgateway
 # exposureClassHandlers:
-# - name: internetconfig
+# - name: internet-config
 #   loadBalancerService:
 #     annotations:
 #       loadbalancer/nework: internet
```
Comment (ialidzhikov) [inline example/20-componentconfig-gardenlet.yaml]:
```suggestion
#       loadbalancer/network: internet
```

File: example/20-componentconfig-gardenlet.yaml [hunk 2]
```diff
@@ -120,11 +120,11 @@ featureGates:
...
-# - name: internalconfig
+# - name: internal-config
 #   loadBalancerService:
 #     annotations:
 #       loadbalancer/nework: internal
```
Comment (ialidzhikov) [inline example/20-componentconfig-gardenlet.yaml]:
```suggestion
#       loadbalancer/network: internet
```

File: example/20-componentconfig-gardenlet.yaml (hunk 1 above)
Comment (vpnachev) [reply example/20-componentconfig-gardenlet.yaml]: Thanks for spotting it!

File: docs/proposals/14-reversed-cluster-vpn.md [hunk 3]
```diff
@@ -197,7 +197,7 @@ With Calico as the networking solution it is not easily possible to put the Wire
...
+Kubelink addresses three challenges managing WireGuard interfaces on cluster nodes. First, with WireGuard interfaces directly on the node (`hostNetwork=true`) the lifecycle of the interface is decoupled from the lifecycle of the pod that created it. This means that there will have to be means of cleaning up the interfaces and its configuration in case the interface moves to a different node. Second, additional routing information must be distributed across the cluster. The WireGuard CIDR is unknown to the network implementation so additional routes must be distributed on all nodes of the cluster. Third, kubelink dynamincally configures the Wireguard interface with endpoints and their public keys.
```
Comment (vpnachev) [inline docs/proposals/14-reversed-cluster-vpn.md:200]: The only actual change is here `hostNework` -> `hostNetwork`.
Rest in this file is trimming trailing white spaces at the end of the lines.

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Review by rfranzke — APPROVED** [review]

**Pull request merged**